src/components/colors/colors.js

```diff
@@ -171,6 +174,9 @@
      * For the evaluated expression, extract the color parts into a hash map
      */
     function extractColorOptions(expression) {
+      if(!expression) {
+        return undefined;
```

Comment:
If this is going to start returning `undefined`, then we need to make sure that callers like `interpolateColors()` on lines 154 and 156 don't call `parseColor(undefined)`.